lib/addCesiumRTC.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var Ellipsoid = Cesium.Ellipsoid;
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+
+module.exports = addCesiumRTC;
+
+function addCesiumRTC(gltf, options) {
+    var positionArray = [];
+    var position = Cartesian3.fromRadians(options.longitude, options.latitude, options.height, Ellipsoid.WGS84);
```

Comment:
Seems easier to just compute the center from the position attribute min/max. It's a little more restrictive, but this makes it easier to add this stage to `gltfPipeline`.